draft-ietf-taps-interface.md

```diff
@@ -1124,6 +1125,30 @@ not all ICMP errors will necessarily be delivered, so applications cannot rely
 on receiving them.
 
 
+### Initiating side is not the first to write {#active-read-before-send}
+
+Name:
+: activeReadBeforeSend
+
+Type:
+: Preference
+
+Default:
+: Ignore
+
```

Comment:
I wonder if this should be `Type: Enumeration`, with values `InitiatorSendsFirst` (the default) and `ListenerSendsFirst`?

Reply:
Why make it so limiting? Preference allows for values such as Prefer, Avoid, Require, ...

Reply:
How does a preference work here? If the application protocol requires the listener to send first, then it need to do this or fail.